### ISSUE TITLE
Quote key moves.

### ISSUE THREAD

**Issue by Gavin-Williams** [GeorgRottensteiner/C64Studio#101]
In the editor, to open double quotes, we use shift+' (windows style ") but after that we have to use the c64 keyboard layout to close quotes, so we use shift+2.

**Comment by GeorgRottensteiner** [comment]
Oooh, good catch. In german keyboard layout that both matches. 
Yeah, it's to enable PETSCII character entry. Of course as a workaround you can turn off the automatic quote mode entry in the preferences.
I'd really rather not lose that entry mode, I'm thinking about actually hard coding the " to work in both modes. Even when it might interfere with custom mapping.

**Comment by Gavin-Williams** [comment]
Yeah, the natural c64 entry mode is a good idea. But maybe dealing with the quotes issue is a good idea too. 

There may be more issues too with regard to the c64 layout, it appears to me that some of the keys, particularly those to the right of the letters are in the wrong position. I have to go and look at my books and at my other programming tool - cbm prg studio - to properly check. You probably already know what I'm talking about.

**Comment by GeorgRottensteiner** [comment]
The default layout mapping is tailored to german keyboards, which layout do you have? If you could modify the BASIC key mapping to your liking, could you export that setting and send it over? Would be nice to include different layouts per default.

**Comment by Gavin-Williams** [comment]
I will, we (my son and I) are just setting up our environment after having a break for a few months, after starting to learn c64 programming earlier in the year. So give me a few days while we get back into the swing of it and I'll do the mapping.

**Comment by GeorgRottensteiner** [comment]
Closing this for now, feel free to re-open!
